.gitignore

```diff
@@ -1,6 +1,5 @@
 bin/*
 *.swp
-pd
```

Comment:
Could you add this back, and maybe also include the `out/*` directory that you now reference in the `Makefile`?